pkg/virt-handler/cgroup/util.go

```diff
@@ -81,18 +84,42 @@ func addCurrentRules(currentRules, newRules []*devices.Rule) ([]*devices.Rule, e
 	return newRules, nil
 }
 
+func getSourceBlockToFsMigratedVolumes(vmi *v1.VirtualMachineInstance, host string) map[string]bool {
```

Comment:
I see we're already pretty clever in the check, looking for the existence of `/dev/volname`.
Would it be easier to do something similar? https://github.com/kubevirt/kubevirt/blob/bb5f94147bdce3c6c75fc0059f3e261b8ca907c1/pkg/virt-handler/cgroup/util.go#L135-L139

Reply:
It wasn't working because the volumes were already filtered out. In the VMI spec they were set to block to fs from the update, and then replaced with hostdisks.

Reply:
@haircommander hey! sorry to bring you in here
we got bitten a few times by replacing the bpf allow list for containers (volume hotplug), because we have to reconstruct the allow list (https://github.com/opencontainers/runc/issues/3196).
do you think it would be realistic to possible to develop an API that allows appending a device to the allow list? I guess on the crun/runc level?

Reply:
I'm not sure that's more of a oci runtime implementer question. I think specifically you're looking for support in libcontainer library right? or do you want an update to the oci runtime spec to allow this?

Reply:
I am not even sure how this would look like tbh. I have a suggestion dropped at the end of that issue for runc but that's about it